AzureFunction/TaskClient.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace Microsoft.Azure.Pipelines.EvaluateArtifactPolicies
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    using Microsoft.Azure.Pipelines.EvaluateArtifactPolicies.Request;
+    using Microsoft.TeamFoundation.DistributedTask.WebApi;
+    using Microsoft.VisualStudio.Services.Common;
+    using Microsoft.VisualStudio.Services.WebApi;
+
+    public class TaskClient : IDisposable
+    {
+        private readonly TaskProperties taskProperties;
+        private TaskHttpClient taskClient;
+        private VssConnection vssConnection;
+        private int? PlanVersion;
+
+        public TaskClient(TaskProperties taskProperties)
+        {
+            this.taskProperties = taskProperties;
+            var vssBasicCredential = new VssBasicCredential(string.Empty, taskProperties.AuthToken);
+            vssConnection = new VssConnection(new Uri(taskProperties.PlanUrl), vssBasicCredential);
+            taskClient = vssConnection.GetClient<TaskHttpClient>();
+        }
+
+        public async Task UpdateTimelineRecordsAsync(TimelineRecord timelineRecord, CancellationToken cancellationToken)
```

Comment:
UpdateTimelineRecordAsync ?

Reply:
Keeping the function names in sync with the client in the package.

Reply:
But it is mis-leading. This method doesn't take an array, it takes one timelineRecord.